source/spawn/spawn-processes-platform__glib.adb

```diff
@@ -324,45 +315,6 @@ package body Spawn.Processes is
       Self.Listener.Standard_Input_Available;
    end Do_Start_Process;
 
-   -----------------
-   -- Environment --
-   -----------------
-
-   function Environment
-     (Self : Process'Class)
-      return Spawn.Environments.Process_Environment is
-   begin
-      return Self.Environment;
-   end Environment;
-
-   ---------------
-   -- Exit_Code --
-   ---------------
-
-   function Exit_Code (Self : Process'Class) return Process_Exit_Code is
-   begin
-      return Self.Exit_Code;
-   end Exit_Code;
-
-   -----------------
-   -- Exit_Status --
-   -----------------
-
-   function Exit_Status (Self : Process'Class) return Process_Exit_Status is
-   begin
-      return Self.Exit_Status;
-   end Exit_Status;
-
-   --------------
-   -- Finalize --
-   --------------
-
-   overriding procedure Finalize (Self : in out Process) is
-   begin
-      --  Shall we check if the process is still running here?
```

Comment:
It could be an issue here, because a "common process" Finalize has:

```ada
      if Self.Status /= Not_Running then
         raise Program_Error;
      end if;
```

So we are changing behavior!

Reply:
It is just stub for implementation, will be added in following patches